packages/aeproject-config/config/config.json

```diff
@@ -16,5 +16,6 @@
         "publicKey": "ak_2mwRmUeYmfuW93ti9HMSUJzCk1EYcQEfikVSzgo6k2VghsWhgU"
     },
     "compilerUrl": "http://localhost:3080",
-    "compilerVersion": "v3.1.0"
+    "compilerVersion": "v3.1.0",
```

Comment:
Don't we need to change the compiler version to v4.0.0 (In addition I am not sure if we are using it anywhere)